.github/workflows/OCV-Coverage-4.x-U20.yaml

```diff
@@ -0,0 +1,295 @@
+name: OCV Coverage:4.x U20
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/OCV-Coverage-4.x-U20.yaml'
+  workflow_call:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 6'
+
+env:
+  EXTRA_CMAKE_OPTIONS: '-DCMAKE_BUILD_TYPE=Debug -DPYTHON_DEFAULT_EXECUTABLE=/usr/bin/python3 -DOPENCV_DOWNLOAD_PATH=/home/ci/binaries_cache -DENABLE_COVERAGE=ON -DBUILD_SHARED_LIBS=ON -DCPU_DISPATCH= -DWITH_IPP=OFF -DBUILD_TESTS=ON -DWITH_OPENNI2=ON -DWITH_GDCM=ON -DCPU_BASELINE=HOST -DENABLE_CCACHE=OFF -DENABLE_PRECOMPILED_HEADERS=OFF -DWITH_GDAL=ON -DBUILD_PERF_TESTS=ON -DBUILD_EXAMPLES=OFF -DBUILD_DOCS=OFF -DWITH_OPENCL=OFF -DWITH_CUDA=OFF -DCPU_BASELINE=SSE,SSE2,SSE3,SSSE3,SSE4.1,POPCNT,SSE4.2,FP16,AVX,AVX2'
```

Comment:
> `-DCPU_BASELINE=SSE,SSE2,SSE3,SSSE3,SSE4.1,POPCNT,SSE4.2,FP16,AVX,AVX2`

`-DCPU_BASELINE=AVX2` should work too.

Reply:
Just wanted to cut `AVX512*`.

Reply:
`-DCPU_BASELINE=AVX2 -DCPU_DISPATCH=` to cut AVX512

Reply:
Changed to you suggestion.